wizard/WizardRestoreWallet4.qml

```diff
@@ -82,11 +82,13 @@ Rectangle {
                     btnNext.enabled = false;
                     wizardController.wizardStateView.wizardRestoreWallet2View.pwField = "";
                     wizardController.wizardStateView.wizardRestoreWallet2View.pwConfirmField = "";
-                    wizardController.recoveryWallet();
-                    wizardController.writeWallet(function() {
-                        wizardController.useMoneroClicked();
-                        btnNext.enabled = true;
-                    });
+                    var recoveryResult = wizardController.recoveryWallet();
+                    if (recoveryResult) {
+                        wizardController.writeWallet(function () {
+                            wizardController.useMoneroClicked();
+                        });
+                    }
+                    btnNext.enabled = true;
```

Comment:
`wizard/WizardCreateWallet5.qml` also uses `btnNext.enabled = true` outside of the `wizardController.writeWallet(...)` scope